source/algorithms.tex

```diff
@@ -2625,20 +2626,20 @@
   }
 
   template<class BidirectionalIterator>
-    constexpr void inplace_merge(BidirectionalIterator first,
+    constexpr void inplace_merge(BidirectionalIterator first,   // hosted
                                  BidirectionalIterator middle,
                                  BidirectionalIterator last);
   template<class BidirectionalIterator, class Compare>
-    constexpr void inplace_merge(BidirectionalIterator first,
+    constexpr void inplace_merge(BidirectionalIterator first,   // hosted
                                  BidirectionalIterator middle,
                                  BidirectionalIterator last, Compare comp);
   template<class ExecutionPolicy, class BidirectionalIterator>
-    void inplace_merge(ExecutionPolicy&& exec,                  // see \ref{algorithms.parallel.overloads}
+    void inplace_merge(ExecutionPolicy&& exec,                  // hosted, see \ref{algorithms.parallel.overloads}
                        BidirectionalIterator first,
                        BidirectionalIterator middle,
                        BidirectionalIterator last);
   template<class ExecutionPolicy, class BidirectionalIterator, class Compare>
-    void inplace_merge(ExecutionPolicy&& exec,                  // see \ref{algorithms.parallel.overloads}
+    void inplace_merge(ExecutionPolicy&& exec,                  // hosted, see \ref{algorithms.parallel.overloads}
```

Comment:
This line is fine, but you need to mark the ranges overloads of `inplace_merge` as hosted.

Reply:
Thank you, fixing!